aiida/orm/nodes/data/cif.py

```diff
@@ -411,7 +411,7 @@ def values(self):
             from CifFile import CifBlock  # pylint: disable=no-name-in-module
 
             with self.open() as handle:
-                c = CifFile.ReadCif(handle, scantype=self.get_attribute('scan_type'))  # pylint: disable=no-member
+                c = CifFile.ReadCif(handle, scantype=self.get_attribute('scan_type', 'standard'))  # pylint: disable=no-member
```

Comment:
Would it be an idea to define `_scan_type_default = 'standard` as a class attribute directly below `_scan_types` and use that here as well as in the constructor?